Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/profil/OwnItemActivity.java

```diff
@@ -0,0 +1,176 @@
+import android.app.SearchManager;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.icu.util.UniversalTimeScale;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.NavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.ActionBarDrawerToggle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.SearchView;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.TextView;
+
+import com.bounswe2017.group10.atlas.R;
+import com.bounswe2017.group10.atlas.httpbody.CultureItem;
+import com.bounswe2017.group10.atlas.httpbody.GetItemsResponse;
+import com.bounswe2017.group10.atlas.profil.ProfileActivity;
+import com.bounswe2017.group10.atlas.response.OnGetItemsResponse;
+import com.bounswe2017.group10.atlas.util.Constants;
+import com.bounswe2017.group10.atlas.httpbody.UserResponse;
+import com.bounswe2017.group10.atlas.remote.APIUtils;
+import com.bounswe2017.group10.atlas.util.Utils;
+import com.bounswe2017.group10.atlas.home.ListItemsFragment;
+import com.bounswe2017.group10.atlas.home.ViewItemFragment;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+import static com.bounswe2017.group10.atlas.util.Utils.getSharedPref;
+import static com.bounswe2017.group10.atlas.util.Utils.logout;
+import static com.bounswe2017.group10.atlas.util.Utils.showToast;
+
+
+public class OwnItemActivity extends AppCompatActivity{
+
+    private ListItemsFragment mFeedFragment;
+    private ListItemsFragment mSearchItemsFragment;
+    private ActionBar mActionBar;
+    private SearchView mSearchView;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_ownitem);
+        Toolbar toolbar = findViewById(R.id.home_toolbar);
+        setSupportActionBar(toolbar);
+        setTitle("My Items");
+        this.mActionBar = getSupportActionBar();
+        mActionBar.setDisplayHomeAsUpEnabled(true);
+
+        mSearchItemsFragment = new ListItemsFragment();
+        setUpSearchFragment();
+
+        mFeedFragment = new ListItemsFragment();
+        setUpFeedFragment();
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // Inflate the menu; this adds items to the action_home bar if it is present.
+        MenuInflater inflater = getMenuInflater();
+        inflater.inflate(R.menu.action_home, menu);
+
+        // Searchable configuration
```

Comment:
Since we don't want search functionality, after expanding ActionBar, hide the search icon.